README.md

```diff
@@ -50,7 +56,9 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
 ```
 
 ### Question 7
-This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code? 
+This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code?
+
+Honestly this is a bit of an educated guess because I'm not qutie sure, if the console.log didn't return "sucess" I would suspect that there is something from with the get request (like that trailing comma after 'GET') but if it did I would say you didng pass an argument to get_all_users.
```

Comment:
You're on the right track in your second guess - `UserApp.add_all_users(data);` needs to be inside of the `.done()` callback in order to have access to the `data` argument.